bundles/org.openhab.transform.jinja/README.md

```diff
@@ -58,12 +58,12 @@ will send the following string out of openHAB
 
 Note that if using \*.things files you must escape quotation marks, for example:
 
-```
+```text
 commandTransformation = "JINJA:{\"msgtype\":\"m.text\", \"body\":\"{{value}}\"}"
 ```
 
 ## Further Reading
 
-- Wikipedia on [Jinja](https://en.wikipedia.org/wiki/Jinja_(template_engine).
+- Wikipedia on [Jinja](<https://en.wikipedia.org/wiki/Jinja_(template_engine)>).
```

Comment:
Why is this necessary?

Reply:
```suggestion
- Wikipedia on [Jinja](https://en.wikipedia.org/wiki/Jinja_(template_engine)).
```

Reply:
Isn't in necessary so that the close parenthesis inside the link isn't interpreted as the end of the link?

Reply:
There are Markdown renders that have issue with both. Maybe you can URL encode the braces it instead?

* [`Wikipedia on [Jinja](<https://en.wikipedia.org/wiki/Jinja_(template_engine)>)`](https://babelmark.github.io/?text=Wikipedia+on+%5BJinja%5D(%3Chttps%3A%2F%2Fen.wikipedia.org%2Fwiki%2FJinja_(template_engine)%3E)
)
* [`Wikipedia on [Jinja](https://en.wikipedia.org/wiki/Jinja_(template_engine))`](https://babelmark.github.io/?text=Wikipedia+on+%5BJinja%5D(https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FJinja_(template_engine)))
* [`Wikipedia on [Jinja](https://en.wikipedia.org/wiki/Jinja_%28template_engine%29)`](https://babelmark.github.io/?text=Wikipedia+on+%5BJinja%5D(https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FJinja_%2528template_engine%2529))